Rakefile

```diff
@@ -106,6 +107,10 @@ namespace :package do
     end
   end
 
+  task :xcode_select do
+    puts `sudo xcode-select --switch #{XCODE_PATH}`
```

Comment:
Should we have some kind of error handling, if the Xcode installation can't be found? Just a simple message telling the user where to put Xcode?